src/controller_main.c

```diff
@@ -209,6 +209,7 @@ void reset_flow_level()
 {
     //reset device state
     CY_Reset_Not_Allow(true);
+    CY_Set_External_Triggered(false);
```

Comment:
Just need to test this change for all flows, in case it has an impact

Reply:
completed